conf/st2.conf.sample

```diff
@@ -59,10 +61,14 @@ api_url = None
 service_token_ttl = 86400
 # Access token ttl in seconds.
 token_ttl = 86400
+# Enable Single Sign On for GUI if true.
+sso = False
 # Authentication mode (proxy,standalone)
 mode = standalone
 # Specify to enable debug mode.
 debug = False
+# Single Sign On backend to use when SSO is enabled. Available backends: noop.
```

Comment:
> Available backends: noop.

This auto-generated part looks non-friendly. In reality, what SSO backends will be available if sso package is installed?

Reply:
The only option is to either manually hard code the list or more this list altogether. The other `saml2` backend is not installed here and `make configgen` here will just keep throwing error.

Reply:
Either that or have a separate sample config for RBAC installations.

Reply:
I manually hard coded this with `noop, saml2`.  It's simpler.

Reply:
:+1: